### ISSUE TITLE
Include user/password for server-status page

### ISSUE THREAD

**Issue by jdvr** [Lusitaniae/apache_exporter#29]
Given the fact that sever-status page could be a security issue configure authentication for this page is necessary and even it is recommended. It would be great to include a user/password parameter and include it as part of the request to the apache server.

**Comment by Lusitaniae** [comment]
Thanks for th suggestion Juan!

Whitelist of the local IP is not enough?

**Comment by jdvr** [comment]
We  think that Whitelist if error prone and may cause a problem on ip reassination after node restarted.

I have been checking the source code and the Go http client and it seems like it is just one line.

What do you think?

**Comment by Lusitaniae** [comment]
I took a look over HaProxy exporter and it seems that it is actually supported by default.
We just need to document it. Would you like to do a PR?

If you want to test yourself, I did the following: 

/etc/apache2/mods-enabled/status.conf 
```
<Location /server-status>
        SetHandler server-status
        AuthType Basic
        AuthName "Restricted Files"
        AuthBasicProvider file
        AuthUserFile "/tmp/apachepasswd"
        Require user prom   
</Location>
```

`
htpasswd -c /tmp/apachepasswd prom
`


`.
/apache_exporter -scrape_uri http://prom:prom@localhost:8888/server-status?auto
`

and it works

**Comment by jdvr** [comment]
Sure, I will finish my task on Monday, so I can test and document it. I will open a PR with the update Readme on Monday. Thanks!

**Comment by Lusitaniae** [comment]
Issue addressed with additional documentation with PR #30.

Thanks for your contribution @jdvr!
